### PR TITLE
Update genesis_file.md

### PR DESCRIPTION
**Pull request by aw126**
Added a few fields “genutil” “gentx” “ibc” “validate-genesis” “evidence”; adjusted consistent dot.

### CODE REVIEW

File: docs/chain-details/genesis_file.md [hunk 1]
```diff
@@ -16,12 +19,13 @@ Specifically, the genesis file includes the following fields:
...
     - `"max_gas"`: The gas limit per block, default value is "-1", i.e., no rules about gas are enforced.
     - `"time_iota_ms"`: The minimum time increment between consecutive blocks, in milliseconds.
-  - `"evidence"`:
+  - `"evidence"`: Evidence storage handling and block proposal detection with the evidence reactor.
     - `"max_age_num_blocks"`: _This field is to be deprecated._
     - `"max_age_duration"`: The maximum age of evidence. Any evidence older than this will be rejected.
     - `"max_num"`: The maximum age of evidence (in number of blocks).
   - `"validator"`:
     - `"pub_key_types"`: The supported validator public key types.
+    - `"validate-genesis"`: Genesis file validation at the default/assigned location.
```
Comment (tomtau) [inline docs/chain-details/genesis_file.md:28]: is `validate-genesis` a field in genesis.json? @lezzokafka

Comment (lezzokafka) [reply docs/chain-details/genesis_file.md:28]: Does not seems like it, `validate-genesis` I know is the command for genesis file validation.

Comment (aw126) [reply docs/chain-details/genesis_file.md]: removed fields that are not in genesis.json

**Review by calvinaco — APPROVED** [review]
LGTM

**Review by lezzokafka — APPROVED** [review]
Thanks @aw126  :D

**Pull request merged**